src/schema/v2/Feature/FeatureImageType.ts

```diff
@@ -34,8 +34,11 @@ export const FeatureImageType = new GraphQLObjectType<
         },
       },
       resolve: ({ image_urls }, args) => {
+        if (!image_urls) return null
```

Comment:
Example of an extremely subtle bug that would have never been found in the pile of sentry logs.

![](http://static.damonzucconi.com/_capture/ihLx3zToLivv.png)

Apparently this field can be completely `null` which is *not* common. Typically this is empty object if there are no images.

Reply:
😮